src/components/main/MovieBar.tsx

```diff
@@ -0,0 +1,22 @@
+import Plus from '@public/svg/plus.svg';
+import Info from '@public/svg/info.svg';
+import RightGo from '@public/svg/rightGo.svg';
+
+export default function MovieBar() {
+  return (
+    <section className="w-full mt-[12px] mb-[22px] flex justify-evenly shrink-0">
+      <div className="flex flex-col items-center basis-12">
+        <Plus className="hover:cursor-pointer" />
+        <p className="text-[14px] leading-[20px]">My List</p>
+      </div>
+      <div className="w-[110px] h-[45px] flex gap-4 items-center justify-center bg-buttonGray rounded-md hover:cursor-pointer hover:bg-buttonHover">
+        <RightGo />
+        <p className="text-[20px] font-[600] text-black">Play</p>
+      </div>
+      <div className="flex flex-col items-center basis-12">
+        <Info className="hover:cursor-pointer" />
+        <p className="text-[14px] leading-[20px]">Info</p>
+      </div>
+    </section>
+  );
+}
```

Comment:
MovieBar 가 세 부분의 컴포넌트로 나뉘어 질 수 있으므로, 각각 MyListButton/ PlayButton / InfoButton 이렇게 3개의 컴포넌트로 생성해 준 뒤, 해당 컴포넌트를 파악하는데 중요한 데이터인 각 버튼의 텍스트를 외부에서 넘겨준다면 가독성에 더 좋을 것 같아요.

```suggestion
   <section className="w-full mt-[12px] mb-[22px] flex justify-evenly shrink-0">
      <MyListButton text="My List" />
      <PlayButton text="Play" />
      <InfoButton text="Info" />
   </section>
```